sku/client.go

```diff
@@ -125,6 +125,10 @@ func (c Client) Update(id string, params *stripe.SKUParams) (*stripe.SKU, error)
 			body.Add("image", params.Image)
 		}
 
+		for k, v := range params.Attrs {
+			body.Add(fmt.Sprintf("attributes[%v]", k), v)
```

Comment:
I'm not sure I 100% follow here, but maybe I'm not giving you enough context around how `attributes` works.

- The keyspace for the `attributes` hash is fixed and determined by the associated product
- Attributes can be partially updated. If I have a SKU with `{attr1: a, attr2: b}`, I can send `{attr1: c}` and the resulting product attributes would be `{attr1: c, attr2: b}`. 
- Attributes cannot be unset. It's not valid to send `{attr1: ""}`. This makes more sense when you think about what attributes actually mean. A shirt can't just lose its size or color, and it's not really well-defined what you'd do with a product that has attributes `size, color` and a SKU with only attribute `color`.

Hope that helps clarify the semantics (: My gut feeling is that this is ~sane, but we should definitely strive to have something simple and obvious here